jtransc-gen-cs/src/com/jtransc/gen/cs/CSharpTarget.kt

```diff
@@ -293,7 +293,11 @@ class CSharpGenerator(injector: Injector) : CommonGenerator(injector) {
 			line("static public void SI()") {
 				val clazzName = if (clazz.isInterface) clazz.name.targetNameForStatic else clazz.name.targetName
 				for (field in clazz.fields.filter { it.isStatic }) {
-					line("$clazzName.${field.targetName} = ${field.escapedConstantValue};")
+					if(field.type==AstType.BOOL){
+						line("$clazzName.${field.targetName} = ${field.escapedConstantValue == "1"};")
```

Comment:
There isnt a Dynamoc.toBool? I cant verify until this evening, but does this works in all cases? Also I would add a test that fails in C# target and to avoid regressions. Maybe it fails in other targets too. Also I think we can do the same in other targets but for doing a safe refactor we will  need a failing case.